packages/eslint-plugin/src/rules/no-context-default-value/index.js

```diff
@@ -0,0 +1,119 @@
+// @ts-check
+const { AST_NODE_TYPES } = require('@typescript-eslint/experimental-utils');
+const createRule = require('../../utils/createRule');
+
+/**
+ * @typedef {import("@typescript-eslint/types/dist/ts-estree").ImportDeclaration} ImportDeclaration
+ * @typedef {import("@typescript-eslint/types/dist/ts-estree").CallExpression} CallExpression
+ * @typedef {import("@typescript-eslint/types/dist/ts-estree").Expression} Expression
+ * @typedef {import("@typescript-eslint/types/dist/ts-estree").Identifier} Identifier
+ * @typedef {import("@typescript-eslint/types/dist/ts-estree").LeftHandSideExpression} LeftHandSideExpression
+ *
+ * @typedef {{
+ *   imports: string[]
+ * }} Options
+ *
+ */
+
+module.exports = createRule({
+  name: 'no-context-default-value',
```

Comment:
please update the README.md and add documentation for this rule so that when the IDE throws an error the squiggly line linke to docs